src/io/PipeReader.zig

```diff
@@ -948,6 +981,7 @@ pub const WindowsBufferedReader = struct {
 
         is_paused: bool = true,
         has_inflight_read: bool = false,
+        use_pread: bool = false,
```

Comment:
8 bits -> 9 bits 😢

Reply:
idk what to do about that, seems fine

Reply:
because the entire struct is 8-byte aligned this doesn't matter anyway